### ISSUE TITLE
Choose a fitting lisence

### ISSUE THREAD

**Issue by stevenvegt** [nuts-foundation/poc-mitz-nuts#1]
CC BY 4.0 Attribution International for content?
https://www.mozilla.org/en-US/MPL/2.0/ for source code?

**Comment by sergejvm** [comment]
Indeed, I would suggest the two licenses @stevenvegt mentions. In GIDS Open Standaarden, we created the following reasoning for those two licenses: https://github.com/GIDSOpenStandaarden/GIDSOpenStandaarden-LegalDocuments/blob/master/Licenties_GIDS.md The documentation also contains good practices for implementing the policies. Comments welcome.
